src/PointwiseFunctions/AnalyticData/Xcts/BinaryWithGravitationalWaves.hpp

```diff
@@ -0,0 +1,627 @@
+      gr::Tags::Conformal<gr::Tags::MomentumDensity<DataType, Dim>, 0> /*meta*/)
+      const;
+  void operator()(
+      const gsl::not_null<Scalar<DataType>*> conformal_factor_minus_one,
+      const gsl::not_null<Cache*> /*cache*/,
+      Xcts::Tags::ConformalFactorMinusOne<DataType> /*meta*/) const;
+  void operator()(
+      const gsl::not_null<Scalar<DataType>*>
+          lapse_times_conformal_factor_minus_one,
+      const gsl::not_null<Cache*> /*cache*/,
+      Xcts::Tags::LapseTimesConformalFactorMinusOne<DataType> /*meta*/) const;
+  void operator()(
+      const gsl::not_null<tnsr::I<DataType, Dim>*> shift_excess,
+      const gsl::not_null<Cache*> /*cache*/,
+      Xcts::Tags::ShiftExcess<DataType, Dim, Frame::Inertial> /*meta*/) const;
+  void operator()(const gsl::not_null<Scalar<DataType>*> rest_mass_density,
+                  const gsl::not_null<Cache*> /*cache*/,
+                  hydro::Tags::RestMassDensity<DataType> /*meta*/) const;
+  void operator()(const gsl::not_null<Scalar<DataType>*> specific_enthalpy,
+                  const gsl::not_null<Cache*> /*cache*/,
+                  hydro::Tags::SpecificEnthalpy<DataType> /*meta*/) const;
+  void operator()(const gsl::not_null<Scalar<DataType>*> pressure,
+                  const gsl::not_null<Cache*> /*cache*/,
+                  hydro::Tags::Pressure<DataType> /*meta*/) const;
+  void operator()(const gsl::not_null<tnsr::I<DataType, 3>*> spatial_velocity,
+                  const gsl::not_null<Cache*> /*cache*/,
+                  hydro::Tags::SpatialVelocity<DataType, 3> /*meta*/) const;
+  void operator()(const gsl::not_null<Scalar<DataType>*> lorentz_factor,
+                  const gsl::not_null<Cache*> /*cache*/,
+                  hydro::Tags::LorentzFactor<DataType> /*meta*/) const;
+  void operator()(const gsl::not_null<tnsr::I<DataType, 3>*> magnetic_field,
+                  const gsl::not_null<Cache*> /*cache*/,
+                  hydro::Tags::MagneticField<DataType, 3> /*meta*/) const;
+
+ private:
+  void add_radiative_term_PN_of_conformal_metric(
+      const gsl::not_null<tnsr::ii<DataType, Dim>*> conformal_metric,
+      const gsl::not_null<Cache*> cache) const;
+  void add_near_zone_term_to_radiative(
+      const gsl::not_null<tnsr::ii<DataType, Dim>*> radiative_term,
+      const gsl::not_null<Cache*> cache) const;
+  void add_present_term_to_radiative(
+      const gsl::not_null<tnsr::ii<DataType, Dim>*> radiative_term,
+      const gsl::not_null<Cache*> cache) const;
+  void add_past_term_to_radiative(
+      const gsl::not_null<tnsr::ii<DataType, Dim>*> radiative_term,
+      const gsl::not_null<Cache*> cache) const;
+  void add_integral_term_to_radiative(
+      const gsl::not_null<tnsr::ii<DataType, Dim>*> radiative_term,
+      const gsl::not_null<Cache*> cache) const;
+};
+
+}  // namespace detail
+
+/*!
+ * \brief   Binary black hole initial data with realistic wave background,
+ * constructed in Post-Newtonian approximations.
```

Comment:
Could you add a list/overview of assumptions that go into this approach, with some notes on limitations and possible plans to deal with them? Specifically: spins, coordinates of the isolated BH solutions, choice of time slicing.

Also, could you add some notes on the goals of this approach, e.g. reduce junk radiation?

Reply:
The time slicing is comming from the trace K which we choose to be the same as in Post-Newtonian ADMTT gauge.
I'll also add the others to the documentation.